src/auto-lightbox.js

```diff
@@ -39,7 +39,7 @@ export const AutoLightboxEvents = {
  */
 export function installAutoLightboxExtension(ampdoc) {
   const {win} = ampdoc;
-  // Only enabled on single documents tagged as <html amp> or <html ⚡>.
+  // Only enabled on single documents tagged as <html amp> or <html ⚡ lang="en">.
```

Comment:
Please remove `lang="en"` from this comment.

Reply:
done